### PR TITLE
Switched from using the requests library to Urllib2

### PR DESCRIPTION
**Pull request by cnuland**
Allow more out of the box usage of the demo project

### CODE REVIEW

File: openshift_gcp.py [hunk 1]
```diff
@@ -56,12 +56,12 @@ def set_openshift_gcp_project_from_google_creds(self):
...
-                headers = { "Metadata-Flavor": "Google" }
-            )
-            r.raise_for_status()
-            self.ocpinv().set_dynamic_cluster_var('openshift_gcp_project', r.text)
+            url = 'http://metadata.google.internal/computeMetadata/v1/project/project-id'
+            headers = { "Metadata-Flavor": "Google" }
+            req = urllib2.Request(url, None, headers)
+            resp = urllib2.urlopen(req)
+            p = resp.read()
+            self.ocpinv().set_dynamic_cluster_var('openshift_gcp_project', p.text)
```
Comment (jkupferer) [inline openshift_gcp.py]: Lines 63+64 should be just:
```
            self.ocpinv().set_dynamic_cluster_var('openshift_gcp_project', resp.read())
```

File: openshift_gcp.py [hunk 2]
```diff
@@ -56,12 +56,12 @@ def set_openshift_gcp_project_from_google_creds(self):
...
         except:
             traceback.print_exec()
```
Comment (jkupferer) [inline openshift_gcp.py]: I spotted an error here too, though it was old code, can you change this from `print_exec` to `print_exc` in your PR?

**Review by jkupferer — CHANGES_REQUESTED** [review]

File: openshift_gcp.py (hunk 1 above)
Comment (cnuland) [reply openshift_gcp.py]: Certainly, cleaned up

File: openshift_gcp.py (hunk 2 above)
Comment (cnuland) [reply openshift_gcp.py]: No problem, done

**Review by jkupferer — APPROVED** [review]

**Pull request merged**